recipes/tree-sitter-glsl/all/conanfile.py

```diff
@@ -0,0 +1,58 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake, cmake_layout
+from conan.tools.files import get, copy
+
+import os
+
+required_conan_version = ">2.0"
+
+
+class TreeSitterCConan(ConanFile):
+    name = "tree-sitter-glsl"
+    description = "GLSL grammar for tree-sitter."
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/tree-sitter-grammars/tree-sitter-glsl"
+    topics = ("parser", "grammar", "tree", "glsl", "ide")
+    settings = "os", "arch", "compiler", "build_type"
+    package_type = "library"
+
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+    implements = ["auto_shared_fpic"]
+    languages = ["C"]
+    exports_sources = "CMakeLists.txt"
+    generators = "CMakeDeps", "CMakeToolchain"
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+    def requirements(self):
+        self.requires("tree-sitter/0.25.1", transitive_headers=True, transitive_libs=True)
```

Comment:
Downgrade `tree-sitter` dependency to be `0.24.3`, the one used along all other grammars in Conan Center Index. This will avoid possible version conflicts in users depending on multiple grammars. 

```suggestion
        self.requires("tree-sitter/0.24.3", transitive_headers=True, transitive_libs=True)
```